packages/common/src/schemas/metadata.ts

```diff
@@ -87,7 +87,7 @@ export const newTrackMetadata = (fields, validate = false): TrackMetadata => {
 const collectionMetadataSchema = {
   is_album: false,
   is_current: true,
-  is_private: true,
```

Comment:
I'm not fully confident in the implications here. I'm assuming this was set to true by default as a precaution, but it seems incorrect since the user should set it via the modal

Reply:
i dont have context on this either, can someone from client chime in?

Reply:
I think we need true here because when you click the create playlist or create album button in the app, we want it to create as hidden by default for you to add tracks and publish later.

I think we should leave this set to `true` and override it in the upload/edit submit based on the form values

Reply:
Hmmm yeah that makes sense. I'll rework this

Reply:
Thought about this some more and I started to do the opposite; i.e. setting `false` by default and making sure the album/playlist buttons were setting it to `true` and it turns out I didn't even need to do anything, it's already explicitly being set.
https://github.com/AudiusProject/audius-protocol/blob/jd/access-and-sale-hidden-disable/packages/web/src/common/store/cache/collections/createPlaylistSaga.ts#L114

Reply:
ok that works for me 👍